doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -104,6 +104,7 @@ Matter
   * The :ref:`ug_matter` protocol page with a table that lists compatibility versions for the |NCS|, the Matter SDK, and the Matter specification.
   * The :ref:`ug_matter_tools` page with installation instructions for the ZAP tool, moved from the :ref:`ug_matter_creating_accessory` page.
   * The :ref:`ug_matter_tools` page with information about CHIP Tool, CHIP Certificate Tool, and the Spake2+ Python tool.
+  * The :ref:`ug_matter_device_low_power_configuration` page with information about ``SLEEPY_ACTIVE_THRESHOLD`` parameter configuration.
```

Comment:
Just noticed, but shouldn't the parameter match the actual one?

Reply:
I'm not quite sure if that's what you mean, but the `SLEEPY_ACTIVE_THRESHOLD` is a parameter name from the Matter specification, that has some default value and we want to allow changing it using the `CONFIG_CHIP_SED_ACTIVE_THRESHOLD` Kconfig.

Reply:
Ah, that makes sense. Should `SLEEPY_ACTIVE_THRESHOLD` then be mentioned in the documentation?

Reply:
We haven't done similar explanations for multiple other terminologies straight from the Matter specification, so I don't want to diverge from it without consulting with Grzegorz. I think that my mistake is that I used this terminology altogether with a style reminding parameter name in the code, so it became confusing. Would it less confusing for you if I rephrase it to the "normal" style?
```suggestion
  * The :ref:`ug_matter_device_low_power_configuration` page with information about sleepy active threshold parameter configuration.
```

Reply:
I think that's a good solution.